examples/demo-complete/demo-complete.sh

```diff
@@ -88,6 +88,7 @@ echo 'output=output/' >> hatchet.ini
 j=$(grep -c ^processor /proc/cpuinfo)
 processes="processes=${j}"
 echo $processes >> hatchet.ini
+echo 'chromosomes="chr22"' >> hatchet.ini
```

Comment:
Specifying `chromosomes` was okay to do in the CI since we only get data for chr22 to avoid huge computation time/disk space. But in this case wasn't the underlying issue here that some steps of hatchet assume that they need to work on certain chromosomes, and later steps violate that assumption by trying to work on more/less chromosomes (chrX/chrY) etc? `chromosomes` was introduced to make this assumption explicit, but things should still work even if its left unspecified. So specifying `chromosomes` here masks the problem instead of solving it?

We don't have to fix any of this as part of the PR of course, and simply making the demo run end-end is a worthwhile effort too - just trying to understand the issue here.